search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -95,3 +105,88 @@
                        "concat humanized rows:" t3))))]
       (info "Create report " t4)
       (str string-writer))))
+
+(defn safe-generate-humanizers-report-csv
```

Comment:
Wait, you mean if the job runs and fails. I don't remember much about the jobs, but we need a way to disable some of the ones that are irrelevant during development. It's really irritating to see all the output from various jobs.